Lesson17/serkan/script.js

```diff
@@ -23,37 +29,53 @@ const findMaxNumberInArray = [-10, -50, -20];
 let maximumNumber = findMaxNumberInArray[0];
 
 for (let i = 1; i < findMaxNumberInArray.length; i++) {
-    if (findMaxNumberInArray[i] > maximumNumber) {
-        maximumNumber = findMaxNumberInArray[i];
-    }
+  if (findMaxNumberInArray[i] > maximumNumber) {
+      maximumNumber = findMaxNumberInArray[i];
+  }
 }
 console.log('Maximum Number In Array is : ' + maximumNumber);
 
 console.log('- - - - - - - - - - - - - - - - - ');
 // 5. (H) Write a loop that finds the minimum number in an array of numbers.
 const findMinNumberInArray = [1, 10, 23, 42, 0, 12, 7, 0];
-
+let minumumNumber = findMinNumberInArray[0];
+for (let i = 1; i < findMaxNumberInArray.length; i++) {
+    if (findMinNumberInArray[i] < minumumNumber) {
+        minumumNumber = findMinNumberInArray[i];
+    }
+}
+console.log('Minimum Number In Array is : ' + minumumNumber);
 console.log('- - - - - - - - - - - - - - - - - ');
 // 6. Write a loop that reverses a given string.
 const string = 'Lorem ipsum dolor sit amet consectetur adipisicing elit.';
 let reversedString = "";
 
 
 for (let i = string.length - 1; i >= 0; i--) {
-  reversedString = reversedString + string[i];
+reversedString = reversedString + string[i];
 }
 
 console.log('The reverse of this string is: ' + reversedString);
 
 console.log('- - - - - - - - - - - - - - - - - ');
 // 7. (H) Write a while loop that counts from 10 to 1 and prints out each number.
+let n=10;
+while(n>0){
+    console.log(n);
+    n++;
```

Comment:
This would be an infinite loop. Since initial value of n is 10, and the condition is valid while n > 0, you need to decrement n.

```suggestion
    n--;
```